### DISCUSSION TITLE
Page title / file name length

### DISCUSSION THREAD

**Discussion by TheGhost78** [gildas-lormeau/SingleFile#1263] [Q&A]
**Is your feature request related to a problem? Please describe.**
Options:
File name:
Template:
{page-title} ({year-locale}-{month-locale}-{day-locale}, {hours-locale}.{minutes-locale}.{seconds-locale}).html

Max length setting: could there be an option for the max length setting to apply only to the {page-title} portion of the file name and not to the file name as a whole? That way I can always have the additional year and time information saved in the file name. Currently, if the page title is really long, the file name truncates before getting to the year and time portions.

**Describe the solution you'd like**
An option for the max length setting to apply only to the {page-title} portion of the file name rather than the file name as a whole.

**Comment by gildas-lormeau** [comment]
This feature already exists, you can append `[n]` after a variable to truncate it to `n` characters/bytes, for example `{page-title}[32]` to keep only the 32 first characters/bytes in the title.
